zone/tradeskills.cpp

```diff
@@ -512,6 +512,15 @@ void Object::HandleAutoCombine(Client* user, const RecipeAutoCombine_Struct* rac
 		return;
 	}
 
+	// Character does not have the required skill.
+	if(spec.skill_needed > 0 && user->GetSkill(spec.tradeskill) < spec.skill_needed ) {
+		// Notify client.
+		user->Message(Chat::Red, "You are not skilled enough.");
```

Comment:
Is there no string ID for this?

Reply:
I will check on the string

Reply:
No specific string. Closest I found is

336 You lacked the skills to fashion %1.

Reply:
Yeah, I don't see one either, closest I found to a confirmation of a message was this: https://forums.daybreakgames.com/eq/index.php?threads/minimum-skill-required.246688/